tests/Event/EventEditingServiceTest.php

```diff
@@ -30,15 +30,22 @@ final class EventEditingServiceTest extends TestCase
      */
     private $readRepository;
 
+    /**
+     * @var DocumentRepository|MockObject
+     */
+    private $organizerDocumentRepository;
```

Comment:
There is no need for a property, the mock creation can be inlined inside the construction of `EventEditingService`